schema/revert/migrations/002_separate_funding_schemas_for_EP_and_IA.sql

```diff
@@ -0,0 +1,7 @@
+-- Revert cif:migrations/002_separate_funding_schemas_for_EP_and_IA from pg
+
+begin;
+
+-- no revert
```

Comment:
I think this one actually is revert-able because we could set all the schema names back to `funding_parameter`, but migration 001 said it's one way so I followed suit